gulpfile.js/import/importRoadmap.js

```diff
@@ -36,8 +38,9 @@ const ROADMAP_DIRECTORY_PATH = utils.project.absolute('pages/shared/data');
 const ALLOWED_ISSUE_TYPES = ['Type: Status Update', 'Status Update'];
 
 // RegEx to extract date from issue title
-const STATUS_UPDATE_REGEX = /(\d*)-(\d*)-(\d*)/;
-
+const STATUS_UPDATE_REGEX = /(\d\d\d\d)-(\d*)-(\d*)/;
+const AMP_COMPONENT_REGEX = /\s(<amp-[\S]*>)/g;
```

Comment:
Nit: could be simplified to `/\s(<amp-\S*>)/g` - please also comment each RegEx individually with an example what it is expected to match.